Tests/AdyenTests/Adyen Tests/Components/Apple Pay/ApplePayComponentTests.swift

```diff
@@ -291,4 +291,12 @@ enum Available {
             return false
         }
     }
+    
```

Comment:
How about we move `Available` enum into it's own file?

Reply:
done 👍